src/AppInstallerCommonCore/Manifest/YamlParser.cpp

```diff
@@ -421,7 +425,8 @@ namespace AppInstaller::Manifest
                 {ManifestInstaller::InstallerSwitchType::Silent, ManifestInstaller::string_t("/quiet")},
                 {ManifestInstaller::InstallerSwitchType::SilentWithProgress, ManifestInstaller::string_t("/passive")},
                 {ManifestInstaller::InstallerSwitchType::Log, ManifestInstaller::string_t("/log \"" + std::string(ARG_TOKEN_LOGPATH) + "\"")},
-                {ManifestInstaller::InstallerSwitchType::InstallLocation, ManifestInstaller::string_t("TARGETDIR=\"" + std::string(ARG_TOKEN_INSTALLPATH) + "\"")}
+                {ManifestInstaller::InstallerSwitchType::InstallLocation, ManifestInstaller::string_t("TARGETDIR=\"" + std::string(ARG_TOKEN_INSTALLPATH) + "\"")},
+                {ManifestInstaller::InstallerSwitchType::Update, ManifestInstaller::string_t("REINSTALL=ALL REINSTALLMODE=vamus")}

```

Comment:
>vamus [](start = 122, length = 5)

Are these the correct flags to use? a means that every file will be rewritten, so it makes me wonder...

Reply:
Chatted offline, we can start with this for most compatible as default

---
In reply to: [499912738](https://github.com/microsoft/winget-cli/pull/588#discussion_r499912738) [](ancestors = 499912738)